provider_azure.go

```diff
@@ -117,6 +119,9 @@ func (a *AzureProvider) getClaimSource(logger log.Logger, allClaims map[string]i
 	if urlParsed.Host == azureADGraphHost {
```

Comment:
Can you add this link to the list here: https://docs.microsoft.com/en-us/graph/migrate-azure-ad-graph-request-differences

Reply:
Added

Reply:
Thank you!